src/object.rs

```diff
@@ -963,6 +963,20 @@ pub struct WeakRef<T: ?Sized> {
     marker: PhantomData<*const T>,
 }
 
+impl<T> WeakRef<T> {
+    pub fn new() -> Self {
```

Comment:
Maybe Default::default instead / in addition?

Reply:
Either would be fine. What would people prefer?